src/constructs/aws/ServerSideWebsite.ts

```diff
@@ -122,15 +122,6 @@ export class ServerSideWebsite extends AwsConstruct {
             queryStringBehavior: OriginRequestQueryStringBehavior.all(),
             headerBehavior: this.headersToForward(),
         });
-        const backendCachePolicy = new CachePolicy(this, "BackendCachePolicy", {
-            cachePolicyName: `${this.provider.stackName}-${id}`,
-            comment: `Cache policy for the ${id} website.`,
-            // For the backend we disable all caching by default
-            defaultTtl: Duration.seconds(0),
-            // Authorization is an exception and must be whitelisted in the Cache Policy
-            // This is the reason why we don't use the managed `CachePolicy.CACHING_DISABLED`
-            headerBehavior: CacheHeaderBehavior.allowList("Authorization"),
```

Comment:
I'm not sure about the impact on forwarding headers because there is this comment

https://github.com/getlift/lift/blob/ee5d84d3a70a879325a651ea2b3f8f497c9b804e/src/constructs/aws/ServerSideWebsite.ts#L380

Reply:
Indeed, this isn't ideal. But related to https://github.com/getlift/lift/issues/144#issuecomment-1142111775 (and the linked Bref issue) that would solve the caching of requests CloudFront receives in the same second.

I don't see another solution.

But at the same time, I don't think `Authorization` is a header that is used a lot for server-side websites?

Also, I haven't even checked if `Authorization` is actually not forwarded when using no cache with CloudFront.

Reply:
I checked this myself and verified that `Authorization` is not forwarded when using the caching disabled policy.

I've had a few cases in the past when I did need the `Authorization` header.

I tested using the custom caching policy with All Cookies and All Query Strings in the cache key, and that prevented same-second requests from hitting cache.